doms-ingester-utils/src/main/java/dk/statsbiblioteket/doms/common/FFProbeParser.java

```diff
@@ -100,7 +118,7 @@ public String getFormatURIFromFFProbeOutput(String FFProbeOutput)
             }
         }
 
-        log.debug("allowedformat: '" + formaturi + "', actual format: '" + format_uri +"'");
+        log.debug("allowedformats: '" + allowedFormats + "', actual format: '" + format_uri +"'");
```

Comment:
Also log formatName here. This log statement is actually quite odd. Think about it